Analysis/Utkscan/processors/source/MtasImplantSipmProcessor.cpp

```diff
@@ -0,0 +1,387 @@
+/** @file MtasProcessor.cpp
+ * @brief  Basic MtasProcessor for MTAS at FRIB
+ * @authors T.T. King, T. Ruland, B.C. Rasco
+ * @date 03/25/2022
+ */
+
+#include "MtasImplantSipmProcessor.hpp"
+
+#include <algorithm>
+#include <cmath>
+#include <cstdlib>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+#include "DammPlotIds.hpp"
+#include "DetectorDriver.hpp"
+#include "DetectorLibrary.hpp"
+#include "Globals.hpp"
+#include "HelperFunctions.hpp"
+#include "RawEvent.hpp"
+#include "StringManipulationFunctions.hpp"
+
+namespace dammIds {
+namespace mtasimplant {
+const unsigned int DD_ANODES_H_ENERGY = 0;
+const unsigned int DD_ANODES_H_OQDC = 1;
+const unsigned int DD_ANODES_H_TQDC = 2;
+const unsigned int DD_ANODES_L_ENERGY = 3;
+const unsigned int DD_ANODES_L_OQDC = 4;
+const unsigned int DD_ANODES_L_TQDC = 5;
+const unsigned int DD_DY_L_ENERGY = 6;
+const unsigned int DD_DY_L_OQDC = 7;
+const unsigned int DD_DY_L_TQDC = 8;
+const unsigned int DD_DY_H_ENERGY = 9;
+const unsigned int DD_DY_H_OQDC = 10;
+const unsigned int DD_DY_H_TQDC = 11;
+const unsigned int DD_SIPM_PIXEL_IMAGE_HG = 12;
+const unsigned int DD_SIPM_PIXEL_IMAGE_LG = 13;
+const unsigned int DD_SIPM_HIRES_IMAGE_LG = 14;
+const unsigned int DD_SIPM_HIRES_IMAGE_HG = 15;
+const unsigned int DD_SIPM_HIRES_IMAGE_LG_QDC = 16;
+const unsigned int DD_SIPM_HIRES_IMAGE_HG_QDC = 17;
```

Comment:
Should be addressed in a different PR/issue as that requires input on how to generate the trace QDC

Reply:
The trace qdc comes from the waveform analyzer. We would need to duplicate the image vectors and some extra math. Not a big deal and is easy to punt to a subsequent PR. Especially since it isn't a workable mode for actual experiments (trace data rates are too much)